wallet.py

```diff
@@ -0,0 +1,69 @@
+import datetime
+from Crypto.Signature import PKCS1_v1_5
+from Crypto.PublicKey import RSA
+
+import os.path
+from pathlib import Path
+
+from blockchain import BlockChain
+from transaction import Transaction, generate_transaction
+ 
+
+if __name__ == '__main__':
+    # os.path.isfile(./wallet.bin)
+    my_file = Path("./wallet.bin")
+    if my_file.is_file():
+        # Open wallet
+        print("Wallet Found")
+        password = input("Enter Password:")
+        with open('walletp.bin','r') as g:
+            private_key = RSA.importKey(g.read(),  passphrase=password)
+            #print(private_key)
+        with open('wallet.bin','r') as f:
+            public_key = f.read()
+
+    else:
+        # Generate wallet keys
+        print("New Wallet Created")
+        password = input("Enter Password:")
+        private_key = RSA.generate(1024)
+        #print(private_key)
```

Comment:
There are a few commented out lines - I'd go ahead and delete them to cleanup the file.